### PR TITLE
Allow setting up ecommerce_worker in devstack.

### PR DESCRIPTION
**Pull request by peter-fogg**
@rlucioni @feanil This PR adds configuration for the ecommerce_worker in devstack. I'm not too familiar with Ansible, so much of this is lifted from the ecommerce service setup.

### CODE REVIEW

File: playbooks/vagrant-devstack.yml [hunk 1]
```diff
@@ -24,6 +24,8 @@
     - elasticsearch
     - forum
     - ecommerce
+    - ecommerce_worker
+    - { role: 'rabbitmq', rabbitmq_ip: '127.0.0.1' }
```
Comment (feanil) [inline playbooks/vagrant-devstack.yml:28]: Why is this explicitly set to 127.0.0.1?

Comment (peter-fogg) [inline playbooks/vagrant-devstack.yml:28]: It's copied from `vagrant-fullstack.yml`. Should it be unset?

**Comment by rlucioni** [comment]
:+1: Please file a support ticket to have values for these variables set in the secure repo.

File: playbooks/vagrant-devstack.yml (hunk 1 above)
Comment (peter-fogg) [inline playbooks/vagrant-devstack.yml:28]: @feanil?

**Comment by feanil** [comment]
:+1:

**Pull request merged**